python/llm/src/bigdl/llm/ggml/convert_model.py

```diff
@@ -0,0 +1,40 @@
+from bigdl.llm.ggml.convert import _convert_to_ggml
+from bigdl.llm.ggml.quantize import quantize
+from pathlib import Path
+import time
+
+
+def convert_model(input_path: str,
+                  output_path: str = None,
+                  model_family: str = 'llama',
+                  dtype: str = 'int4'):
+    """
+    Convert Hugging Face llama-like / gpt-neox-like / bloom-like model to lower precision
+
+    :param input_path: str, path of model, for example `./llama-7b-hf`.
+    :param output_path: Save path of output quantized model. Default to `None`.
+            If you don't specify this parameter, quantized model will be saved in
+            the same directory as the input and just replace precision with quantize_type
+            like `./ggml-model-q4_0.bin`.
```

Comment:
I don't think we should support `None`; we may support model meta info in future @TheaperDeng ?

Reply:
yeah a good way for future is to require a dir path here.